aw_watcher_window/printAppStatus.jxa

```diff
@@ -23,37 +23,73 @@ var url = undefined, incognito = undefined, title = undefined;
 switch(appName) {
   case "Safari":
     // incognito is not available via safari applescript
+    try { // see catch for details
     url = Application(appName).documents[0].url();
-    title = Application(appName).documents[0].name();
+	  title = Application(appName).documents[0].name();
```

Comment:
Please try to keep whitespace consistent (2 spaces for indentation, no end-of-line spaces)